src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpReplIdeFeatures.cs

```diff
@@ -0,0 +1,346 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Shared.TestHooks;
+using Microsoft.VisualStudio.IntegrationTest.Utilities;
+using Roslyn.VisualStudio.IntegrationTests.Extensions.Interactive;
+using Xunit;
+
+namespace Roslyn.VisualStudio.IntegrationTests.CSharp
+{
+    [Collection(nameof(SharedIntegrationHostFixture))]
+    public class CSharpReplIdeFeatures : AbstractInteractiveWindowTest
+    {
+        public CSharpReplIdeFeatures(VisualStudioInstanceFactory instanceFactory)
+            : base(instanceFactory)
+        {
+            VisualStudioWorkspaceOutOfProc.SetUseSuggestionMode(true);
+        }
+
+        public new void Dispose()
+        {
+            VisualStudioWorkspaceOutOfProc.SetUseSuggestionMode(false);
+            Reset();
+            base.Dispose();
+        }
+
+        // https://github.com/dotnet/roslyn/issues/801
+        // [Fact]
+        public void VerifyDefaultUsingStatements()
+        {
+            SubmitText("Console.WriteLine(42);");
+            this.WaitForLastReplOutput("42");
+        }
+
+        // https://github.com/dotnet/roslyn/issues/801
+        // [Fact]
+        public void VerifyCodeActionsAvailableInCurrentSubmission()
+        {
+            InsertCode("Directory.Exists(\"foo\");");
+            //      <VerifyCodeActions ApplyFix = "using System.IO;" >
```

Comment:
Thank you! 
1. I migrated some more disabled tests.
2. There are 2 tests with "XML". Both seem to be reasonable to keep but not reasonable to migrate right now.
3. Now I migrated all tests with CodeActions. However, they are disabled due to open issues. I can verify CodeActions scenarios in standalone VisualStudio but not in integration tests. Will sync with Jonathon tomorrow.

---
In reply to: [108558613](https://github.com/dotnet/roslyn/pull/18137#discussion_r108558613) [](ancestors = 108558613)